### PR TITLE
adjust CI dev image tags

### PR DESCRIPTION
**Pull request by wanlonghenry**
adjust CI dev image tags

### CODE REVIEW

**Comment by wanlonghenry** [comment]
Triggered build with the same changes  both from automated and manually:
image
image
image

File: .pipelines/azure_pipeline_mergedbranches.yaml [hunk 1]
```diff
@@ -33,11 +33,12 @@ jobs:
...
-      commit=$(git describe) 
-      linuxImagetag="$commit"
-      windowsImageTag=win-"$commit"
+      commit=$(git describe | cut -d'-' -f1,2)
+      datetime=$(date +%Y%m%d%H%M%S)
+      linuxImagetag="$commit"-"$datetime"
+      windowsImageTag=win-"$commit"-"$datetime"
 
-      telemetryTag=$commit
+      telemetryTag=$linuxImagetag
```
Comment (jatakiajanvi12) [inline .pipelines/azure_pipeline_mergedbranches.yaml:41]: I think we should keep both the commit id and datetime.

File: .pipelines/azure_pipeline_mergedbranches.yaml [hunk 2]
```diff
@@ -33,11 +33,12 @@ jobs:
   ## prod release are in format Major.Minor.Patch
   ## preview release are in format preview-Major.Minor.Patch
   - bash: |
-      commit=$(git describe) 
-      linuxImagetag="$commit"
-      windowsImageTag=win-"$commit"
+      commit=$(git describe | cut -d'-' -f1,2)
```
Comment (ganga1980) [inline .pipelines/azure_pipeline_mergedbranches.yaml]: why are we removing the commit id? we can keep commit id and add timestamp onto that, right?

Comment (wanlonghenry) [reply .pipelines/azure_pipeline_mergedbranches.yaml]: updated, keep both id and date

File: .pipelines/azure_pipeline_mergedbranches.yaml (hunk 1 above)
Comment (wanlonghenry) [reply .pipelines/azure_pipeline_mergedbranches.yaml:41]: updated, keep both id and date

**Comment by wanlonghenry** [comment]
triggered the build with same change:
image
image

**Review by ganga1980 — APPROVED** [review]

**Review by jatakiajanvi12 — APPROVED** [review]

**Pull request merged**